docs/guide/install.md

```diff
@@ -202,7 +203,8 @@ Options:
 - `--range` (`string`): page ranges to export (example: `'1,4-5,6'`).
 - `--dark` (`boolean`, default: `false`): export as dark theme.
 - `--with-clicks`, `-c` (`boolean`, default: `false`): export pages for every click animation (see https://sli.dev/guide/animations.html#click-animations).
-- `--theme`, `-t` (`string`): override theme.
+- `--theme` (`string`): override theme.
+- `--template`, `-t` (`string`, default: `default`): specify [the print template](/guide/exporting#print-template).
```

Comment:
I would prefer `--type` with hard-coded options like `slides` `notes` `handout` - and let ppl to override them instead of creating arbitrary options. Because we might want to do some special handling for each type - over-generalize it might not be ideal for long term.

Reply:
I think it is OK to allow users to create new print templates. This is because it will be the user/addon's responsibility to handle other options like `--with-clicks` or so.

About the name of this option, should we change all "print templates" into "print types" or only the CLI flag?